plugin-server/src/utils/db/db.ts

```diff
@@ -746,6 +746,7 @@ export class DB {
                 GROUP BY team_id, id
                 HAVING max(is_deleted)=0
             )
+            ORDER BY created_at
```

Comment:
I'm not sure about this - it could make the query slower in prod, instead lets order in `getEventsByPerson` in the tests at the end

Reply:
that function is only used in tests (we definitely don't want to pull the whole persons table on prod!), but I changed the PR to sort persons in `getEventsByPerson` instead, to reduce the scope of change.

Reply:
oh I see. .. maybe that should belong to a test/utils/db.ts or something similar file then